src/utilities/filetypes/EpwFile.cpp

```diff
@@ -3967,6 +3974,31 @@ bool EpwFile::parse(std::istream& ifs, bool storeData) {
     }
   }
 
+  for (unsigned int i = 0; i < epw_strings.size(); i++) {
+    int year = epw_strings[i].year;
+    int month = epw_strings[i].month;
+    int day = epw_strings[i].day;
+    int hour = epw_strings[i].hour;
+    int currentMinute = epw_strings[i].currentMinute;
```

Comment:
Making these copies seems superfluous. Perhaps you could just use `epw_string.currentMinute` when you need to access these members. At minimum they could be const.

Reply:
Moving all `epw_string.month`, `epw_string.day`, etc to where they are used below.